service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapperTest.java

```diff
@@ -29,10 +29,7 @@
 import org.mockito.quality.Strictness;
 import org.mockito.stubbing.Answer;
 
```

Comment:
The imports look to have be auto re-ordered by itellij, probably best we keep them in the same order to make PR's with fewer changes